DashAI/back/api/api_v1/endpoints/datasets.py

```diff
@@ -77,6 +85,70 @@ async def get_dataset(dataset_id: int, db: Session = Depends(get_db)):
     return dataset
 
 
+@router.get("/sample/{dataset_id}")
+async def get_sample(dataset_id: int, db: Session = Depends(get_db)):
+    """Return the dataset with id dataset_id from the database.
+
+    Parameters
+    ----------
+    dataset_id : int
+        id of the dataset to query.
+
+    Returns
+    -------
+    Dict
+        A Dict with a sample of 10 rows
+    """
+    try:
+        file_path = db.get(Dataset, dataset_id).file_path
+        dataset: DashAIDataset = load_dataset(f"{file_path}/dataset")
+        sample = dataset["train"].sample(n=10)
+        if not file_path:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail="Dataset not found",
+            )
```

Comment:
The  `if not file_path`  condition must be just after the line 103. 
Otherwise, if the file does not exists, `load_dataset` could raise an exception.

```suggestion
        file_path = db.get(Dataset, dataset_id).file_path
        if not file_path:
            raise HTTPException(
                status_code=status.HTTP_404_NOT_FOUND,
                detail="Dataset not found",
            )
        dataset: DashAIDataset = load_dataset(f"{file_path}/dataset")
        sample = dataset["train"].sample(n=10)

```